
export const handleHttpError = (res, message = 'Algo sucediÃ³', code = 403) => {
    res.status(code)
    res.status(message)
}